mobile/lib/main.dart

```diff
@@ -500,5 +503,34 @@ Future<void> runBackend(bridge.Config config) async {
 
   FLog.info(text: "App data will be stored in: $appDir");
   FLog.info(text: "Seed data will be stored in: $seedDir");
+
+  await initFirebase();
+  await requestNotificationPermission();
+
   await rust.api.runInFlutter(config: config, appDir: appDir, seedDir: seedDir);
 }
+
+Future<void> initFirebase() async {
+  await Firebase.initializeApp(
+    options: DefaultFirebaseOptions.currentPlatform,
+  );
+}
+
+/// Ask the user for permission to send notifications via Firebase
+Future<void> requestNotificationPermission() async {
+  FirebaseMessaging messaging = FirebaseMessaging.instance;
+  final token = await messaging.getToken();
+  FLog.info(text: "Firebase token: $token");
```

Comment:
Nit: I don't think this deserves an `info` log level.

Reply:
Guide for Android testers: @bonomat and @Restioson - please DM me your firebase tokens for your Android devices when you deploy, and then I can send you some notifications before the release.

You can do the same from the Firebase Console as well yourselves if you prefer to. I will send a guide today.